data-raw/scenario_geography_x_ald_sector.R

```diff
@@ -1,100 +1,137 @@
-                               "Japan",      "Power",   "WEO2020_SDS",
-                               "Japan",      "Power",   "WEO2020_SPS",
-                        "LatinAmerica",       "Coal",   "WEO2020_SDS",
-                        "LatinAmerica",    "Oil&Gas",   "WEO2020_SDS",
-                        "LatinAmerica",      "Power",   "WEO2020_SDS",
-                        "LatinAmerica",       "Coal",   "WEO2020_SPS",
-                        "LatinAmerica",    "Oil&Gas",   "WEO2020_SPS",
-                        "LatinAmerica",      "Power",   "WEO2020_SPS",
-                          "MiddleEast",       "Coal",   "WEO2020_SDS",
-                          "MiddleEast",    "Oil&Gas",   "WEO2020_SDS",
-                          "MiddleEast",      "Power",   "WEO2020_SDS",
-                          "MiddleEast",       "Coal",   "WEO2020_SPS",
-                          "MiddleEast",    "Oil&Gas",   "WEO2020_SPS",
-                          "MiddleEast",      "Power",   "WEO2020_SPS",
-                             "NonOECD",       "Coal",   "WEO2020_SDS",
-                             "NonOECD",    "Oil&Gas",   "WEO2020_SDS",
-                             "NonOECD",      "Power",   "WEO2020_SDS",
-                             "NonOECD",       "Coal",   "WEO2020_SPS",
-                             "NonOECD",    "Oil&Gas",   "WEO2020_SPS",
-                             "NonOECD",      "Power",   "WEO2020_SPS",
-                        "NorthAmerica",       "Coal",   "WEO2020_SDS",
-                        "NorthAmerica",    "Oil&Gas",   "WEO2020_SDS",
-                        "NorthAmerica",      "Power",   "WEO2020_SDS",
-                        "NorthAmerica",       "Coal",   "WEO2020_SPS",
-                        "NorthAmerica",    "Oil&Gas",   "WEO2020_SPS",
-                        "NorthAmerica",      "Power",   "WEO2020_SPS",
-                                "OECD",       "Coal",   "WEO2020_SDS",
-                                "OECD",    "Oil&Gas",   "WEO2020_SDS",
-                                "OECD",      "Power",   "WEO2020_SDS",
-                                "OECD",       "Coal",   "WEO2020_SPS",
-                                "OECD",    "Oil&Gas",   "WEO2020_SPS",
-                                "OECD",      "Power",   "WEO2020_SPS",
-                              "Russia",      "Power",   "WEO2020_SDS",
-                              "Russia",      "Power",   "WEO2020_SPS",
-                         "SouthAfrica",      "Power",   "WEO2020_SDS",
-                         "SouthAfrica",      "Power",   "WEO2020_SPS",
-                       "SoutheastAsia",      "Power",   "WEO2020_SDS",
-                       "SoutheastAsia",      "Power",   "WEO2020_SPS",
-                                  "US",      "Power",   "WEO2020_SDS",
-                                  "US",      "Power",   "WEO2020_SPS"
-   )
+  ~scenario_geography,  ~ald_sector,          ~scenario,
```

Comment:
you add rows to scenario_geography_x_ald_sector here but one can see that the respective rda file gets smaller. This looks fishy, please check

Reply:
i hope my comment makes sense rda and R are in line and as stated above only 2021 instead of 2019 2020